pkg/aks/components/__tests__/CruAks.test.ts

```diff
@@ -1,13 +1,15 @@
 import semver from 'semver';
 import flushPromises from 'flush-promises';
-import { shallowMount, Wrapper } from '@vue/test-utils';
+import { shallowMount, Wrapper, config, mount } from '@vue/test-utils';
 import CruAks from '@pkg/aks/components/CruAks.vue';
 // eslint-disable-next-line jest/no-mocks-import
 import { mockRegions, mockVersionsSorted } from '../../util/__mocks__/aks';
 import { AKSNodePool } from 'types';
 import { _EDIT, _CREATE } from '@shell/config/query-params';
 import { nodePoolNames } from '../../util/validators';
 
+config.global.renderStubDefaultSlot = true;
```

Comment:
I thought I had written a comment here but I did not.
Should this config not be defined within the mock of the specific test?
I am not so sure if this is affecting the whole configuration or not. If we need it global we could even set it into Jest.

Reply:
I've removed this now. It's actually not needed as I took a different approach to handling the default slot breaking change.